packages/maskbook/src/components/InjectedComponents/NFT/NFTAvatar.tsx

```diff
@@ -0,0 +1,288 @@
+                                          )}
+                                      />
+                                  ))}
+                        </Box>
+
+                        {!(page === 0 && collectibles_.length === 0) ? (
+                            <TablePagination
+                                count={-1}
+                                component="div"
+                                onPageChange={() => {}}
+                                page={page}
+                                rowsPerPage={30}
+                                rowsPerPageOptions={[30]}
+                                labelDisplayedRows={() => null}
+                                backIconButtonProps={{
+                                    onClick: () => setPage(page - 1),
+                                    size: 'small',
+                                    disabled: page === 0,
+                                }}
+                                nextIconButtonProps={{
+                                    onClick: () => setPage(page + 1),
+                                    disabled: !hasNextPage,
+                                    size: 'small',
+                                }}
+                            />
+                        ) : null}
+                        <Box className={classes.button}>
+                            <Button
+                                variant="contained"
+                                size="medium"
+                                className={classes.setNFTAvatar}
+                                onClick={() => onClick()}
+                                disabled={!selectedToken}>
+                                {t('profile_nft_avatar_set')}
+                            </Button>
+                        </Box>
+                    </Box>
+                </EthereumChainBoundary>
+            </Box>
+            <AddNFT open={open_} onClose={() => setOpen_(false)} onAddClick={onAddClick} />
+        </>
+    )
+}
+
+export interface AvatarMetaDB {
+    userId: string
+    tokenId: string
+    amount: string
+    image?: string
+    name?: string
+    address: string
+    avatarId?: string
+    symbol: string
+}
+
+export async function saveNFTAvatar(userId: string, avatarId: string, contract: string, tokenId: string) {
+    const asset = await PluginCollectibleRPC.getAsset(contract, tokenId)
+    const orders =
+        !asset.sellOrders || asset.sellOrders?.length === 0
+            ? !asset.orders || asset.orders?.length === 0
+                ? !asset.buyOrders || asset.buyOrders?.length === 0
+                    ? []
+                    : asset.buyOrders
+                : asset.orders
+            : asset.sellOrders
+
+    const order = head(
+        orders.sort(
+            (a, b) =>
+                new BigNumber(getOrderUnitPrice(a) ?? 0).toNumber() -
+                new BigNumber(getOrderUnitPrice(b) ?? 0).toNumber(),
```

Comment:
Compare with `Number` method `isGreaterThan` or `gt`

Reply:
isGreaterThan  returns  bool

Reply:
> isGreaterThan returns bool

Then you should `BigNumber#minus` and `BigNumber#toNumber`

Reply:
minus is ok